tools/downloader/list_topologies.yml

```diff
@@ -1,3480 +0,0 @@
-    files:
-      - name: license-plate-recognition-barrier-0007.tar.gz
-        sha256: 6c0b8651f0eead6de43e07491fd41809a4a7da141f1b5002dfd8dd207bdb1018
-        source: https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/models/lpr/chinese_lp/license-plate-recognition-barrier-0007.tar.gz
-    output: "optical_character_recognition/license_plate_recognition/tf/"
-    postprocessing:
-      - $type: unpack_archive
-        format: gztar
-        file: license-plate-recognition-barrier-0007.tar.gz
-    model_optimizer_args:
-      - --framework=tf
-      - --data_type=FP32
-      - --reverse_input_channels
-      - --input_shape=[1,24,94,3]
-      - --input=input
-      - --scale_values=input[254.99997577500233]
-      - --output=d_predictions
-      - --input_model=$dl_dir/license-plate-recognition-barrier-0007/graph.pb.frozen
-    framework: tf
-    license: https://raw.githubusercontent.com/opencv/training_toolbox_tensorflow/develop/LICENSE
-
-  - name: "face-detection-retail-0044"
-    description: "Face Detection (SqNet1.0modif+single scale) with BatchNormalization trained with negatives"
-    task_type: "detection"
-    files:
-      - name: face-detection-retail-0044.prototxt
-        sha256: 89ea5bb9c293ebabb56b9f6dfb4cfc6c46cd146cdce307a2f5a2755bf904fc79
-        source: https://raw.githubusercontent.com/opencv/training_toolbox_caffe/63ccbbc328f0c1f414f459c284293b3929b09339/models/face_detection/deploy.prototxt
-      - name: face-detection-retail-0044.caffemodel
-        sha256: 10f68b73f3b68db89fab151be51fc90a0bc1e2a18a87ac642e88aa9e94d53f8f
-        source: https://github.com/opencv/training_toolbox_caffe/raw/63ccbbc328f0c1f414f459c284293b3929b09339/models/face_detection/face-detection-retail-0044.caffemodel
-    output: "Retail/object_detection/face/sqnet1.0modif-ssd/0044/caffe/"
-    model_optimizer_args:
-      - --framework=caffe
-      - --data_type=FP32
-      - --input_shape=[1,3,300,300]
-      - --input=data
-      - --input_model=$dl_dir/face-detection-retail-0044.caffemodel
-      - --input_proto=$dl_dir/face-detection-retail-0044.prototxt
-    framework: caffe
-    license: https://raw.githubusercontent.com/opencv/training_toolbox_caffe/develop/LICENSE
-
-  - name: "mobilenet-v1-1.0-224"
-    description: >-
-      `mobilenet-v1-1.0-224` is one of MobileNet V1 architecture <https://arxiv.org/abs/1704.04861>
-      with the width multiplier 1.0 and resolution 224. It is small, low-latency,
-      low-power models parameterized to meet the resource constraints of a variety
-      of use cases. They can be built upon for classification, detection, embeddings
-      and segmentation similar to how other popular large scale models are used.
-    task_type: "classification"
-    files:
-      - name: mobilenet-v1-1.0-224.prototxt
-        sha256: 8e6a26b8f2c7cf7a066d571660cfd8a1544a5a25399df33ad499f3d733f16729
-        source: https://raw.githubusercontent.com/shicai/MobileNet-Caffe/26a8b8c0afb6114a07
```

Comment:
+please resolve conflict

Reply:
I will re-split the config before merging.